packages/history/index.ts

```diff
@@ -215,17 +193,6 @@ export interface History {
    * @see https://github.com/remix-run/history/tree/main/docs/api-reference.md#history.go
    */
   go(delta: number): void;
-
-  /**
-   * Sets up a listener that will be called whenever the current location
-   * changes.
-   *
-   * @param listener - A function that will be called when the location changes
-   * @returns unlisten - A function that may be used to stop listening
-   *
-   * @see https://github.com/remix-run/history/tree/main/docs/api-reference.md#history.listen
-   */
-  listen(listener: Listener): () => void;
```

Comment:
Got rid of this for now - I don't think there's any need for it in `MemoryHistory` since there's no `onpopstate`?  I think if we limit this to be a single listener in `BrowserHistory`, it may make sense to go with an `onPopState` callback on the factory function rather than an imperative `listen` method

Reply:
I think `window.history.go(-1)` triggers `window.onpopstate` so it'll need to in memory history, too.